src/mom5/ocean_csiro_bgc/csiro_bgc.F90

```diff
@@ -1018,10 +1021,12 @@ subroutine csiro_bgc_sbc(isc, iec, jsc, jec, isd, ied, jsd, jed, &
 
 do j = jsc, jec  !{
   do i = isc, iec  !{
-    sc_o2(i,j) = 1638.0 + t_prog(indtemp)%field(i,j,1,time%taum1) *  &
-         (-81.83 + t_prog(indtemp)%field(i,j,1,time%taum1) *         &
-          (1.483 + t_prog(indtemp)%field(i,j,1,time%taum1) *         &
-           (-0.008004))) * grid%tmask(i,j,1)
+    ! Limit temp used for Schmidt number calculation.
+    schmidt_temp = min(35.0, t_prog(indtemp)%field(i,j,1,time%taum1))
+    sc_o2(i,j) = 1638.0 + schmidt_temp * &
+        (-81.83 + schmidt_temp *         &
+        (1.483 + schmidt_temp *          &
+        (-0.008004))) * grid%tmask(i,j,1)
```

Comment:
I don't expect this to be fixed, but felt duty bound to say "wow there are a lot of magic numbers here".

Reply:
Alakazam